daemon/graphdriver/proxy.go

```diff
@@ -35,6 +37,11 @@ type graphDriverInitRequest struct {
 }
 
 func (d *graphDriverProxy) Init(home string, opts []string) error {
+	if !d.p.IsV1() {
+		if cp, ok := d.p.(plugingetter.CountedPlugin); ok {
+			cp.Acquire()
```

Comment:
Hm, I see you're doing the same for cleanup. Perhaps add a comment to explain is by design